# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/clients/manager/ClientIPCTypes.ipdlh
# Commit: 3399e7c51942
# Full Hash: 3399e7c519424a82824f48abd3b2a4d75346d723
# Author: Christoph Kerschbaumer <ckerschb@christophkerschbaumer.com>
# Date: 2019-05-22 03:40:58
# Regressor Bug: 965637
# File Overlap Count: 3
# Description:
#   Bug 965637: Move CSP from Principal into Client, part 2: worker changes. r=baku
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D27655
# ==============================================================================

diff -r ddf4012a7652 -r 3399e7c51942 dom/clients/manager/ClientIPCTypes.ipdlh
--- a/dom/clients/manager/ClientIPCTypes.ipdlh	Tue May 21 23:14:27 2019 +0000
+++ b/dom/clients/manager/ClientIPCTypes.ipdlh	Tue May 21 23:14:50 2019 +0000
@@ -110,7 +110,7 @@
 struct ClientOpenWindowArgs
 {
   PrincipalInfo principalInfo;
-  ContentSecurityPolicy[] cspInfos;
+  CSPInfo? cspInfo;
   nsCString url;
   nsCString baseURL;
 };